airflow/contrib/kubernetes/kubernetes_request_factory/pod_request_factory.py

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 import yaml
-from airflow.contrib.kubernetes.pod import Pod
+from airflow.contrib.kubernetes.pod import Pod # noqa
```

Comment:
??

Reply:
If I run flake locally, this warning is reported. `airflow.contrib.kubernetes.pod` is reported as unused. The only use of this class is in the comment, which contains information about the types. https://github.com/apache/airflow/blob/2d3a9232f62a118f782fe5329cb595fff5e7f22e/airflow/contrib/kubernetes/kubernetes_request_factory/pod_request_factory.py#L44 I am going to rewrite it into a different style of comments that does not cause such problems.

Reply:
The format of the comment needs to stay as it is to support typing on Python3 but not break with Python2 doesn't it?

Maybe I didn't see the flake failure as I'm on Python3.

The other option may be to put behind a guard like this:

```
if TYPE_CHECKING:
     from airflow.contrib.kubernetes.pod import Pod
```

Reply:
I was already changing so that it would be uniform throughout the source code.

Reply:
I withdrew these changes finally. This does not apply to these PR.